app/players/dwarves/dwarf_blacksmith.py

```diff
@@ -0,0 +1,17 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfBlacksmith(Dwarf):
+    def __init__(
+            self, nickname: str, favourite_dish: str, skill_level: int
+    ) -> None:
+        super().__init__(favourite_dish)
+        self.nickname = nickname
+        self.__skill_level = skill_level
+
+    def get_rating(self) -> int:
+        return self.__skill_level
+
+    def player_info(self) -> str:
+        return f"Dwarf blacksmith {self.nickname}" \
+               f" with skill of the {self.__skill_level} level"
```

Comment:
Better to use () instead of \
```suggestion
        return (
            f"Dwarf blacksmith {self.nickname}" 
            f" with skill of the {self.__skill_level} level"
        )
```

Reply:
Fix in all places